### ISSUE TITLE
Dark mode does not work on Android with Expo (managed)

### ISSUE THREAD

**Issue by assimovt** [react-native-datetimepicker/datetimepicker#523]
# Bug report

## Summary

The dark mode does not work on Android.

## Reproducible sample code

Here is a minimal repo with similar basic usage as described in the README:
https://github.com/assimovt/expo-datetime-example

## Steps to reproduce

- Open the demo app in Android and set the dark mode in the Settings
- Click "SHOW DATE PICKER!"
- According to the [RNTimePickerDialogFragment.java#L81-L87](https://github.com/react-native-datetimepicker/datetimepicker/blob/v3.5.2/android/src/main/java/com/reactcommunity/rndatetimepicker/RNTimePickerDialogFragment.java#L81-L87) the dark theme should be picked up from the OS, but it's not.

## Environment info

**datetimepicker version: 3.5.2**

```info Fetching system and libraries information...
System:
    OS: macOS 11.2.3
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 381.55 MB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.15.1 - ~/.nvm/versions/node/v14.15.1/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 7.5.4 - ~/.nvm/versions/node/v14.15.1/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.10.1 - /Users/me/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.5, DriverKit 20.4, macOS 11.3, tvOS 14.5, watchOS 7.4
    Android SDK: Not Found
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.7199119
    Xcode: 12.5.1/12E507 - /usr/bin/xcodebuild
  Languages:
    Java: 16.0.2 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz => 0.63.2 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

Android version: SDK Android 11.0 Simulator
Expo version: SDK 42 (managed)

**Comment by jereztech** [comment]
+1

**Comment by vonovak** [comment]
:tada: This issue has been resolved in version 6.3.2 :tada:

If this package helps you, consider [sponsoring us](https://github.com/sponsors/react-native-datetimepicker)! :rocket:

**Comment by sharathbangera6** [comment]
Is this fixed ? i dont see it working in 6.7.3. Haven't changed much in the code. 

app.json
"userInterfaceStyle": "automatic"

Is there anything else i need to do ?
